File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Fastperiode.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;


@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
public class Fastperiode {


    private String sluttdato;

    private String startdato;

    public String getSluttdato() {
        return sluttdato;
    }

    public void setSluttdato(String sluttdato) {
        this.sluttdato = sluttdato;
    }

    public String getStartdato() {
        return startdato;
    }

    public void setStartdato(String startdato) {
        this.startdato = startdato;
    }

}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Fastperiode.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Fastperiode (
    var sluttdato: String? = null,
    var startdato: String? = null
)
